App/Components/ProgressiveImage.tsx

```diff
@@ -12,7 +12,61 @@ export interface ProgressiveImageProps {
   style?: StyleProp<ImageStyle>
 }
 
-export default class ProgressiveImage extends React.Component<ProgressiveImageProps> {
+export interface ProgressiveImageState {
+  androidPreview: boolean
+}
+
+
+export default class ProgressiveImage extends React.Component<ProgressiveImageProps, ProgressiveImageState> {
+  constructor (props: ProgressiveImageProps) {
+    super(props)
+    this.state  = {
+      androidPreview: true
+    }
+  }
+
+  _androidLoad () {
+    this.setState({androidPreview: false})
+  }
+
+  android () {
+    return <Fragment>
+      <TextileImage
+        imageId={this.props.imageId}
+        path={'thumb'}
+        style={[{position: 'absolute', top: 0, bottom: 0, left: 0, right: 0}, this.props.style]}
+        resizeMode={this.props.resizeMode}
+        capInsets={this.props.capInsets}
+      />
+      <TextileImage
+        imageId={this.props.imageId}
+        path={this.props.path}
+        style={[{backgroundColor: 'transparent', position: 'absolute', top: 0, bottom: 0, left: 0, right: 0}, this.props.style, this.state.androidPreview && {height: 0}]}
+        resizeMode={this.props.resizeMode}
+        capInsets={this.props.capInsets}
+        onLoad={this._androidLoad.bind(this)}
+      />
+    </Fragment>
```

Comment:
Android native textile image element wasn't setup to allow them to be nested like ios does. so went with the less time consuming fix here for now. works well enough.